menu-animation/index.html

```diff
@@ -0,0 +1,69 @@
+          />
+        </svg>
+        <div class="menu-item__title">
+          <div class="menu-item__title-value">Events</div>
+        </div>
+      </div>
+      <div class="menu-item">
+        <svg viewBox="0 0 512 512" class="menu-item__icon">
+          <path
+            fill="currentColor"
+            d="M452 40h-24V0h-40v40H124V0H84v40H60C27 40 0 67 0 100v352c0 33 27 60 60 60h392c33 0 60-27 60-60V100c0-33-27-60-60-60zm20 412c0 11-9 20-20 20H60c-11 0-20-9-20-20V188h432v264zm0-304H40v-48c0-11 9-20 20-20h24v40h40V80h264v40h40V80h24c11 0 20 9 20 20v48z"
+          />
+          <path
+            d="M76 230h40v40H76zM156 230h40v40h-40zM236 230h40v40h-40zM316 230h40v40h-40zM396 230h40v40h-40zM76 310h40v40H76zM156 310h40v40h-40zM236 310h40v40h-40zM316 310h40v40h-40zM76 390h40v40H76zM156 390h40v40h-40zM236 390h40v40h-40zM316 390h40v40h-40zM396 310h40v40h-40z"
+          />
+        </svg>
+        <div class="menu-item__title">
+          <div class="menu-item__title-value">Search</div>
+        </div>
+      </div>
+      <div class="menu-item">
+        <svg viewBox="0 0 512 512" class="menu-item__icon">
+          <path
+            fill="currentColor"
+            d="M452 40h-24V0h-40v40H124V0H84v40H60C27 40 0 67 0 100v352c0 33 27 60 60 60h392c33 0 60-27 60-60V100c0-33-27-60-60-60zm20 412c0 11-9 20-20 20H60c-11 0-20-9-20-20V188h432v264zm0-304H40v-48c0-11 9-20 20-20h24v40h40V80h264v40h40V80h24c11 0 20 9 20 20v48z"
+          />
+          <path
+            d="M76 230h40v40H76zM156 230h40v40h-40zM236 230h40v40h-40zM316 230h40v40h-40zM396 230h40v40h-40zM76 310h40v40H76zM156 310h40v40h-40zM236 310h40v40h-40zM316 310h40v40h-40zM76 390h40v40H76zM156 390h40v40h-40zM236 390h40v40h-40zM316 390h40v40h-40zM396 310h40v40h-40z"
+          />
+        </svg>
+        <div class="menu-item__title">
+          <div class="menu-item__title-value">Updates</div>
+        </div>
+      </div>
+      <div class="menu-item">
+        <svg viewBox="0 0 512 512" class="menu-item__icon">
+          <path
+            fill="currentColor"
+            d="M452 40h-24V0h-40v40H124V0H84v40H60C27 40 0 67 0 100v352c0 33 27 60 60 60h392c33 0 60-27 60-60V100c0-33-27-60-60-60zm20 412c0 11-9 20-20 20H60c-11 0-20-9-20-20V188h432v264zm0-304H40v-48c0-11 9-20 20-20h24v40h40V80h264v40h40V80h24c11 0 20 9 20 20v48z"
+          />
+          <path
+            d="M76 230h40v40H76zM156 230h40v40h-40zM236 230h40v40h-40zM316 230h40v40h-40zM396 230h40v40h-40zM76 310h40v40H76zM156 310h40v40h-40zM236 310h40v40h-40zM316 310h40v40h-40zM76 390h40v40H76zM156 390h40v40h-40zM236 390h40v40h-40zM316 390h40v40h-40zM396 310h40v40h-40z"
+          />
+        </svg>
+        <div class="menu-item__title">
+          <div class="menu-item__title-value">Settings</div>
+        </div>
```

Comment:
In real world these elements would be interactive. To go to page for example, Maybe we can use something better than `divs`? :)

Reply:
Yes, I think it would be better to use `a` here :) And if it is a nav user should have a chance to open it by mouse wheel, or by context menu